docs/r2/tensorboard_in_notebooks.ipynb

```diff
@@ -0,0 +1,491 @@
+            "Epoch 5/5\n",
+            "60000/60000 [==============================] - 12s 197us/sample - loss: 0.3128 - accuracy: 0.8840 - val_loss: 0.3668 - val_accuracy: 0.8617\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "metadata": {
+        "id": "SxvXc4hoKW7d",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "Start TensorBoard within the notebook using [magics](https://ipython.readthedocs.io/en/stable/interactive/magics.html):"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "KBHp6M_zgjp4",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "%tensorboard --logdir logs"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "Po7rTfQswAMT",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "<img class=\"tfo-display-only-on-site\" src=\"images/notebook_tensorboard.png?raw=1\"/>"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "aQq3UHgmLBpC",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "You can now view dashboards such as scalars, graphs, histograms, and others. Some dashboards are not available yet in Colab (such as the profile plugin).\n",
+        "\n",
+        "Note that the format is very similar to how TensorBoard would be used from the command line."
+      ]
+    },
+    {
+      "metadata": {
+        "id": "NiIMwOG8MR_g",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "You can also start TensorBoard before training to monitor it in progress:"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "qyI5lrXoMw9K",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "%tensorboard --logdir logs"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "ALxC8BbWWV91",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "<img class=\"tfo-display-only-on-site\" src=\"images/notebook_tensorboard_two_runs.png?raw=1\"/>"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "GUSM8yLrO2yZ",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "The same TensorBoard backend is reused by issuing the same command. If a different logs directory was chosen, a new instance of TensorBoard would be opened. Ports are managed automatically. \n",
+        "\n",
+        "Start training a new model and watch TensorBoard update automatically every 30 seconds or refresh it with the button on the top right:"
```

Comment:
It might be nicer to continue training the existing model rather than
restarting from scratch, so that the user can see the metrics continue
to improve instead of seeing two new curves basically superimposed on
the first ones (up to random jitter). Is this feasible?

Reply:
It is possible, we have the technology. I'll make this change.

Reply:
It turns out that while it is possible to continue training, the TensorBoard charts don't zoom out to view the new data, so the user won't see any updates unless they click on "fit domain to data". 

The following chart has 10 epochs (the last 5 are part of the "continued training)":

![image](https://user-images.githubusercontent.com/2601900/53663164-ea0bc680-3c19-11e9-9bb8-6711fa3eb847.png)

Reply:
Of course; forgot about that. Thanks.

Reply:
Opened #1946 to track this. We can close if it's a dupe of an existing issue.

Reply:
Extra thanks—I don’t think that it’s a dupe (I looked for a while, but
forgot to actually open the issue myself!).